h2o-py/h2o/schemas/error.py

```diff
@@ -37,7 +38,16 @@ def __str__(self):
             if self.payload[1]: res += "    json: %r\n" % self.payload[1]
             if self.payload[2]: res += "    file: %r\n" % self.payload[2]
             if self.payload[3]: res += "    params: %r\n" % self.payload[3]
+        if self.show_stacktrace and self.stacktrace:
+            res += "  Stacktrace: %s\n" % self._format_stacktrace(indent=6)
         return res
+    
+    def _format_stacktrace(self, indent=4, top=10):
```

Comment:
It seems that H2OModelBuilderErrorV3 does a similar thing - should this code be unified?

Reply:
Right, it was my original intent, but looks like I forgot then… will use this function in `H2OModelBuilderErrorV3`

Reply:
for `H2OModelBuilderErrorV3` I'll render the entire stacktrace though, mainly useful if we have some cases of wrapped exceptions.